com.github.akiraux.akira.json

```diff
@@ -0,0 +1,61 @@
+{
+   "app-id": "com.github.akiraux.akira",
+   "runtime": "org.gnome.Platform",
+   "runtime-version": "3.36",
+   "sdk": "org.gnome.Sdk",
+   "base":"io.elementary.BaseApp",
+   "base-version": "juno-19.08",
+   "command": "com.github.akiraux.akira",
+   "cleanup": [
+      "/include",
+      "/lib/pkgconfig",
+      "/lib/debug",
+      "/share/vala",
+      "/man",
+      "*.a",
+      "*.la"
+   ],
+   "finish-args": [
+      "--share=ipc",
+      "--socket=fallback-x11",
+      "--socket=wayland",
+      "--talk-name=org.gtk.vfs.*",
+      /* dconf */
```

Comment:
you shouldn't need this dconf snippet in apps based on 19.08 or 3.36 runtimes or later. Please remove.